src/app/data/static/utils.ts

```diff
@@ -209,4 +209,16 @@ export class Utils {
       return 'https://etherscan.io';
     }
   }
+
+  public static prettyTransactionType(type: string): string {
```

Comment:
When I said about enum I mean you must do this like this:
`transactions-types.constant.ts`:
```
export enum TransactionsTypes {
  BUY: 'Buy',
  SELL: 'Sell',
  ADD: 'Add liquidity',
  REM: 'Remove liquidity',
  CONVERSION: 'Conversion',
}
```

then you can use it like this:
```
  || uniDto?.type === TransactionsTypes.REM
  || uniDto?.type === TransactionsTypes.ADD
```
at etc.